streams/src/main/java/org/apache/kafka/streams/processor/internals/SinkNode.java

```diff
@@ -58,8 +60,21 @@ public void addChild(final ProcessorNode<Void, Void, ?, ?> child) {
     public void init(final InternalProcessorContext<Void, Void> context) {
         super.init(context);
         this.context = context;
-        keySerializer = prepareKeySerializer(keySerializer, context, this.name());
-        valSerializer = prepareValueSerializer(valSerializer, context, this.name());
+        try {
+            keySerializer = prepareKeySerializer(keySerializer, context, this.name());
+        } catch (final ConfigException e) {
+            throw new ConfigException(String.format("Failed to initialize key serdes for sink node %s", name()));
```

Comment:
Should we somehow preserve `e.getMessage()` -- it seems useful?

Reply:
added original message